File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex15.java
//Importando Scanner
import java.util.Scanner;

public class Ex15 {

	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	double salarioInicial, conta1, conta2, totalPagar, salarioRestante;
	
	System.out.println("Salário restante\n\n");
	
	//Entrada
	System.out.println("Insira o salário: R$ ");
	salarioInicial = input.nextFloat();
	
	System.out.print("Insira o valor da 1° conta: R$ ");
	conta1 = input.nextDouble();
	
	System.out.print("Insira o valor da 2° conta: R$ ");
	conta2 = input.nextDouble();
		
	//Processamento
	totalPagar = ( conta1 + conta2 ) * 1.02;
	salarioRestante = salarioInicial - totalPagar;
	
	//Saída
	System.out.println("Total a pagar: R$ " + totalPagar);
	System.out.printf("Salário restante: R$ %.2f", salarioRestante);
	
		
	}
}

File: 3 semestre/Estrutura de Dados I/Listas/Lista de Matrizes/ex15.c
#include <stdio.h>

main()
{
    /* vars */
    int m[5][5], i, j;

    printf("MATRIZ IDENTIDADE (5 x 5)\n");

    /* Entrada, Processamento e Saída */
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            if (i == j) {
                m[i][j] = 1;
            } else {
                m[i][j] = 0;
            }

            printf("%d  ", m[i][j]);
        }
        printf("\n");
    }

}
